documentation/releaseProcess.md

```diff
@@ -0,0 +1,40 @@
+# Release process
+
+Production releases for libraries within va-mobile-library are currently
+triggered manually by engineers after each merge into the `main` branch. We have
+a [Publish Workflow in GitHub Actions](https://github.com/department-of-veterans-affairs/va-mobile-library/actions/workflows/publish.yml) 
+which automatically increments the specified package's version, creates a git 
+tag, and publishes a new version to NPM. 
+
+Alpha and beta releases are also released using the same workflow, but are 
+typically triggered from a development branch. See the [Versioning Policy](./versioning.md) 
```

Comment:
Don't think these links will work within the doc site, other pages have used the full URLs.